README.md

```diff
@@ -127,6 +127,8 @@ preview(DateRange)                   | Object    |                  | displays a
 showPreview(DateRange)               | bool      | true             | visibility of preview
 editableDateInputs(Calendar)         | bool      | false            | whether dates can be edited in the Calendar's input fields
 dragSelectionEnabled(Calendar)       | bool      | true             | whether dates can be selected via drag n drop
+calendarFocus(Calendar)              | String    | 'forwards'       | Whether calendar focus month should be forward-driven or backwards-driven. can be 'forwards' or 'backwards'
+preventUnnecessaryRefocus(Calendar)  | bool      | false            | prevents unneceessary refocus of shown range on selection
```

Comment:
`Unnecessary` int he prop name seems redundant and I think we can rename the prop to exclude that so it is more concise.
wdyt?

Reply:
Thanks for the feedback! How about something more like `preventSnapRefocus`?

Reply:
sure, that looks better :)